test/tools/Modules/HelpersCommon/HelpersCommon.psm1

```diff
@@ -385,3 +385,19 @@ function Get-PlatformInfo {
         return "unknown"
     }
 }
+
+# return true if WsMan is supported on the current platform
+function Get-WsManSupport {
+    $platformInfo = Get-PlatformInfo
+    if (
+        ($platformInfo.Platform -match "alpine|raspbian") -or
+        ($platformInfo.Platform -eq "debian" -and ($platformInfo.Version -eq '10' -or $platformInfo.Version -eq '')) -or # debian 11 has empty Version ID
```

Comment:
Debian 11 platform version == '' is not true anymore:

```
PS /> cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

Reply:
```suggestion
        ($platformInfo.Platform -eq "debian" -and ($platformInfo.Version -ne '9') -or 
```